publisher_joint_trajectory_planned:
  ros__parameters:

    controller_name: "joint_trajectory_controller"
    wait_sec_between_publish: 1

    joints:
      - arm_shoulder_lift_joint
      - arm_shoulder_pan_joint
      - arm_elbow_joint
      - arm_wrist_1_joint
      - arm_wrist_2_joint
      - arm_wrist_3_joint

    check_starting_point: false
    starting_point_limits:
      # shoulder_pan_joint: [-0.1,0.1]
      # shoulder_lift_joint: [-1.6,-1.5]
      # elbow_joint: [-0.1,0.1]
      # wrist_1_joint: [-1.6,-1.5]
      # wrist_2_joint: [-0.1,0.1]
      # wrist_3_joint: [-0.1,0.1]
      arm_shoulder_pan_joint: [-0.1,0.1]
      arm_shoulder_lift_joint: [-1.3,-1.1]
      arm_elbow_joint: [-2.4,-2.2]
      arm_wrist_1_joint: [-1.3,-1.1]
      arm_wrist_2_joint: [1.5,1.6]
      arm_wrist_3_joint: [2.7,2.9]
